Plugin.cs

```diff
@@ -122,11 +122,21 @@ public static void AddReputationPoints_PrePatch(ReputationService __instance, fl
             if (!__instance.IsValidReputationGain(amount) || cheatConfig == null || Serviceable.BuildingsService.Seals.Count > 0)
                 return;
 
+            var biomeName = Serviceable.BiomeService.CurrentBiome.Name;
+            WriteLog(biomeName);
+            WriteLog(biomeName);
+            WriteLog(biomeName);
             var newAmount = amount * cheatConfig.ReputationMutiplier;
+            // it'd be nice if we could figure out when an archaeology dig site is available based off the buildings list
+            //var maxReputation = Serviceable.BuildingsService.Relics.Count == 0
+            //    ? (float)__instance.GetReputationToWin() - cheatConfig.ReputationStopgap
+            //    : (float)__instance.GetReputationToWin() - cheatConfig.ReputationStopgap - 1;
             // rep stopgaps should change a bit based on events in the map, such as archaeologist ruins
-            var maxReputation = Serviceable.BuildingsService.Altars.Count == 0
+            // no I don't like hardcoded strings, but it'll have to do for now...
+            var maxReputation = Serviceable.BiomeService.CurrentBiome.Name != "Scarlet Orchard"
```

Comment:
Best practice would be to use ToUpperInvariant and compare it to "SCARLET ORCHARD"

Reply:
Good catch, thx